account_invoice_under_payment/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Invoice Under Payment.
```

Comment:
Module title shoul be written as follows:

```
==============
{module_title}
==============
```
Please see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst as reference.